kedro-telemetry/kedro_telemetry/plugin.py

```diff
@@ -99,28 +115,90 @@ def before_command_run(
             )
 
 
-def _format_user_cli_data(
-    hashed_username: str, command_args: List[str], project_metadata: ProjectMetadata
+class KedroTelemetryHooks:  # pylint: disable=too-few-public-methods
+    """Hook to send proejct statistics data to Heap"""
+
+    project_properties = PROJECT_PROPERTIES if PROJECT_PROPERTIES else {}
```

Comment:
Do we need the `if else` statement here? If the `PROJECT_PROPERTIES` haven't been populated by a CLI call it's just an empty dict anyway right?